wrappers/python/simtk/openmm/__init__.py

```diff
@@ -20,6 +20,8 @@
 
 from simtk.openmm.openmm import *
 from simtk.openmm.vec3 import Vec3
+from mtsintegrator import MTSIntegrator
+from amd import AMDIntegrator, AMDForceGroupIntegrator, DualAMDIntegrator
```

Comment:
The relative import works fine.  I don't know why the two lines above it are using absolute.  There's no particular need for it.